cli/lib/kontena/cli/certificate/authorize_command.rb

```diff
@@ -1,22 +1,45 @@
+require_relative '../services/services_helper'
 
 module Kontena::Cli::Certificate
   class AuthorizeCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
+    include Kontena::Cli::Services::ServicesHelper
 
 
     parameter "DOMAIN", "Domain to authorize"
 
+    option '--type', 'AUTHORIZATION_TYPE', 'Authorization type, either tls-sni-01 or dns-01', default: 'dns-01'
+    option '--linked-service', "LINKED_SERVICE", 'A service (usually LB) where the tls-sni-01 certificate is bundled to'
```

Comment:
Where the `tls-sni-01` **challenge** certificate... I see the potential for some confusion re deploying the actual service cert here

Reply:
good point